doc/internal/man3/ossl_provider_new.pod

```diff
@@ -199,8 +199,6 @@ passed back to some of the provider functions.
 
 ossl_provider_forall_loaded() iterates over all the currently
 "activated" providers, and calls I<cb> for each of them.
-If no providers have been "activated" yet, it tries to activate all
```

Comment:
Why was this removed??

Reply:
Because it is a lie.  The code doesn't do this.

I've no idea why the code doesn't do it.

Reply:
Isn't that what the `provider_activate_fallbacks` call does?